specs/data_structures.md

```diff
@@ -112,6 +113,7 @@ Block header, which is fully downloaded by both full clients and light clients.
 | `lastBlockID`                     | [BlockID](#blockid)       | Previous block's ID.                                                                                                                                               |
 | `lastCommitHash`                  | [HashDigest](#hashdigest) | Previous block's Tendermint commit hash.                                                                                                                           |
 | `consensusRoot`                   | [HashDigest](#hashdigest) | Merkle root of [consensus parameters](#consensus-parameters) for this block.                                                                                       |
+| `feeHeader`                       | [FeeHeader](#feeheader)   | Header data pertaining to fees.                                                                                                                                    |
```

Comment:
The rationale to not include this in the header is because it isn't data light clients would need to care about?

Reply:
Oh no. It's abstracted like this in case we want to e.g. not do fee burning. Then the fields pertaining to fees can be swapped out without affecting everything else.